STEERING-COMMITTEE.md

```diff
@@ -77,12 +80,6 @@ options.
 Seats on the Steering Committee are held by an organization, not by the
 individual.
 
-The committee was created as the project was in its infancy, in order to tackle
-governance and overall project strategy. Because of the nature of the project
-and funding required, it was decided that strong corporate leadership was
-necessary for the project to ensure velocity. As the project grows and matures
-the KSC will, from time to time, consider if this policy should be changed.
-
```

Comment:
This removal seems to foreclose the idea of later governance changes, which seems limiting.

Reply:
I actually agree with the removal, I don't think this document is the right place for historical context. However it should lay out clearly how to amend the governance.